src/raft_server.cxx

```diff
@@ -1151,7 +1151,7 @@ void raft_server::check_leadership_transfer() {
         if (peer_elem->get_matched_idx() + params->stale_log_gap_ <
                 cur_commit_idx) {
             // This peer is lagging behind.
-            p_tr("peer %d is lagging behind, %lu < %lu",
+            p_tr("peer %d is lagging behind, %llu < %llu",
```

Comment:
Please use `PRIu64`. This symbol is different according to the platform.

Reply:
There are many %lu, %llu, %d use case across the whole repo, but I am only fixing this line because it has warning during compile. 

I fixed it to use PRIu64.

Reply:
`%d` probably be ok, but 64-bit integers should be replaced. The whole projected was re-visited a couple of years ago, but if there are still missing ones, we need to fix them.

Reply:
```
~ grep -rn "%l" .                                                                               
./fix_format.hxx:20:    replace_all(format_string, "%ld", "%" PRId64); \
./fix_format.hxx:21:    replace_all(format_string, "%lu", "%" PRIu64); \
./asio_service.cxx:336:                      "due to error %d, %s, ref count %ld",
```

We don't need to care about ./fix_format.hxx, the %ld in ./asio_service.cxx is correct, I think that is all the case.